### DISCUSSION TITLE
Pagination

### DISCUSSION THREAD

**Discussion by suaveone** [livewire/livewire#2573] [4. General]
I found that if I add the pagination links twice, say top and bottom of a large list of data, the pagination links break. Unless I am doing something wrong.
![Screen Shot 2021-02-26 at 5 02 07 PM](https://user-images.githubusercontent.com/30648644/109370312-a40f8a80-7854-11eb-8fb2-2ae16bb0ab81.png)

**Comment by joshhanley** [comment]
@suaveone check out this thread and this particular comment for how you might be able to work around it https://github.com/livewire/livewire/issues/1582#issuecomment-727070911

Hope this helps!
